src/pages/index.js

```diff
@@ -83,8 +106,12 @@ profileEditButton.addEventListener("click", function () {
 });
 
 function handleEditFormSubmit(data) {
-  userInfo.setUserInfo(data);
-  profileEditPopup.close();
+  return api
+    .updateUserInfo({ name: data.title, about: data.description })
+    .then(() => {
+      userInfo.setUserInfo(data);
+      profileEditPopup.close();
```

Comment:
`      profileEditPopup.close();
`

You close the modal inside `PopupWithForm`, so you don't need to duplicate it in `index.js`


**Please, correct it everywhere in the code**